File: is28/shishmakova28/Python/zad6/prog1.py
def massivslovare():
    f=open('file6.txt','r')
    a=[]
    v={}
    for i in range(2):
        buf = f.readline()
        buf1 = buf.split(':')
        v['name']=buf1[0]
        v['surname']=buf1[1]
        v['age']=int(buf1[2])
        a.append(v.copy())
    return a

vyvod=massivslovare()
print(vyvod)

File: is28/pavlov28/C++/lab2/prog1.cpp
#include <string>
#include <iostream>

using namespace std;

int main()
{
	float age;
	float kurs;
	string name;

	cout << "Как вас зовyт? ";
	cin >> name;
	cout << "Привет, " << name << endl;

	cout << "Сколько вам лет? ";
	cin >> age;

	cout << "На каком вы кyрсе? ";
	cin >> kurs;

	cout << "Вы закончите ВYЗ в " << age+4-kurs << endl;
	return 0;
}
